Plots/Streamlines/NCL_stream_1.py

```diff
@@ -0,0 +1,88 @@
+"""
+NCL_stream_1.py
+===============
+Basic streamlines drawing
+
+Concepts illustrated:
+   - Drawing a black-and-white streamline plot over a map
+
+Replicates the NCL script: https://www.ncl.ucar.edu/Applications/Scripts/stream_1.ncl
+"""
+
+################################################################################
+#
+# import modules
+#
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy.feature as cfeature
+import geocat.viz as gcv
+import geocat.datafiles
+
+
+################################################################################
+#
+# open data file and extract variables. We extract a 2D horzontal slice from
+# the first time step of the 3D U and V variables at the at the bottom level.
+#
+ds = xr.open_dataset(geocat.datafiles.get('netcdf_files/uvt.nc'))
+
+timestep = 0
+level = 0
+U = ds.U[timestep,level,:,:]
+V = ds.V[timestep,level,:,:]
```

Comment:
`ds.U.isel(time=0, lev=0)` and  `ds.V.isel(time=0, lev=0)` are preferable since they use appropriate dimension names and do not require knowing which axis of the array `time` and/or `lev` are occupying.

Reply:
Are 'lev' and 'time' simply whatever dimension names are used in the NetCDF file, or does Xarray abstract the temporal and vertical dimension so that they can always be identified as 'time' and 'lev'? Thanks!

Reply:
> Are 'lev' and 'time' simply whatever dimension names are used in the NetCDF file

Yes

Reply:
Too bad. Maybe a future hackathon topic :-)

@andersy005 this is ready for re-review. Thanks.